nav2_bringup/launch/cloned_multi_tb3_simulation_launch.py

```diff
@@ -24,20 +24,43 @@
     AppendEnvironmentVariable,
     DeclareLaunchArgument,
     ExecuteProcess,
+    ForEach,
     GroupAction,
     IncludeLaunchDescription,
     LogInfo,
     OpaqueFunction,
     RegisterEventHandler,
 )
+
 from launch.conditions import IfCondition
 from launch.event_handlers import OnShutdown
+from launch.launch_context import LaunchContext
 from launch.launch_description_sources import PythonLaunchDescriptionSource
 from launch.substitutions import LaunchConfiguration, TextSubstitution
-from nav2_common.launch import ParseMultiRobotPose
+
+import yaml
+
+
+def count_robots(context: LaunchContext) -> list[LogInfo]:
```

Comment:
It's kind of unfortunate that you have to do this and parse the config string yourself. What if the `ForEach` action itself logged the number of iterations?

Reply:
That would be great!